sql/catalyst/src/test/scala/org/apache/spark/sql/RowTest.scala

```diff
@@ -121,10 +121,6 @@ class RowTest extends FunSpec with Matchers {
       externalRow should be theSameInstanceAs externalRow.copy()
     }
 
-    it("copy should return same ref for internal rows") {
```

Comment:
Well `MutableRow` is mutable, so it shouldn't hold for those. The only exception is `GenericInternalRow`.

That being said, I don't if you remove/modify the test.